gym_minigrid/roomgrid.py

```diff
@@ -72,8 +72,8 @@ def __init__(
         num_rows=3,
         num_cols=3,
         max_steps=100,
-        seed=0,
```

Comment:
This looks more like a bug that the environment defaults to a fixed seed, but I just want to check

Reply:
I am not sure there ever has been an instantiation of `RoomGrid` without specifying the seed. It does not matter anymore actually, given that seeding is handled by the `reset` function.

Reply:
This probably matter to add to the release notes

Reply:
What does this mean I should do ? (Sorry, I'm not very familiar with all the terms)